### ISSUE TITLE
Horizontal scrolling of code on iPhone not working

### ISSUE THREAD

**Issue by jvandertil** [jvandertil/blog#34]
On iOS 13.3.1 horizontal scrolling of code blocks does not work

**Comment by jvandertil** [comment]
![DE5B6060-A824-4EE6-9241-D38B4E74B777](https://user-images.githubusercontent.com/158879/73478781-0159b380-4397-11ea-811d-6acaab16e908.jpeg)

As an example

**Comment by jvandertil** [comment]
Resource that seems to work: https://benfrain.com/horizontal-scrolling-area-css-overflow-ios/

**Comment by jvandertil** [comment]
Fixed by adding the following CSS:
```css
.highlight {
    -webkit-overflow-scrolling: touch;
}

.highlight > pre > code {
    display: inline-block;
}
```

The `-webkit-overflow-scrolling` is not strictly necessary but gives a nice scroll bar.

Debugged using ngrok, [this](https://jsonhilder.github.io/blog/ngrok-hugo/) blogpost was handy for setting up ngrok.

In short:
```sh
.\hugo serve -DEF --minify -b https://<id>.ngrok.io --bind 127.0.0.1 --port 1313 --appendPort=false
```

**Comment by jvandertil** [comment]
Fixed with 05309a577766b5a27736b36f7f17d0ff5617e992
